CHANGELOG.md

```diff
@@ -41,7 +41,7 @@
 **Fixed:**
 
 - bpk-component-scrollable-calendar: 4.0.25 => 5.0.0
-  - Fixed a reintroduced functionality of auto scrolling the calendar when a date is clicked, when the library was changed from `react-window` to `react-virtualized` 
+  - Fixed a reintroduced functionality of auto scrolling the calendar when a date is clicked, when the library was changed from `react-window` to `react-virtualized`
```

Comment:
I guess the whitespace changes have been done because `trim_trailing_whitespace = true` is set in `.editorconfig` and my editor picked it.
I've left the changes because I think it makes sense to keep the file aligned to the settings, but happy to revert it if you want :-)